### PR TITLE
SAK-48641 - Profile image doesn't work if font libraries aren't available on the system

### PR DESCRIPTION
**Pull request by jonespm**
Add more logging to help determine that fonts are the problem

### CODE REVIEW

**Review by mpellicer — APPROVED** [review]

File: entitybroker/rest/src/java/org/sakaiproject/entitybroker/rest/EntityActionsManager.java [hunk 1]
```diff
@@ -227,6 +230,7 @@ public ActionReturn handleCustomActionExecution(ActionsExecutable actionProvider
                     } else if (e.getCause().getClass().isAssignableFrom(SecurityException.class)) {
                         throw new SecurityException(e.getCause().getMessage() + " (rethrown)", e.getCause());
                     }
+                    log.error("Exception {} not rethrown, throwing RuntimeException.", e.getCause().getMessage());
```
Comment (ern) [inline entitybroker/rest/src/java/org/sakaiproject/entitybroker/rest/EntityActionsManager.java:233]:
```suggestion
                    log.error("Target exception class not identified, re throwing as a RuntimeException: {}", e.toString());
```

Comment (ern) [reply entitybroker/rest/src/java/org/sakaiproject/entitybroker/rest/EntityActionsManager.java:233]: minor suggestion where e.toString() will also include the class name in the message which can be very useful in these situations!

Comment (ern) [reply entitybroker/rest/src/java/org/sakaiproject/entitybroker/rest/EntityActionsManager.java:233]: `toString()` is simply more informative than `getMessage()` and really should be used almost always when logging the reason

```
    public String toString() {
        String s = getClass().getName();
        String message = getLocalizedMessage();
        return (message != null) ? (s + ": " + message) : s;
    }
```

**Comment by jonespm** [comment]
Okay thanks for the additional information, it looks like rethrowing is often just a single word though so committed without the space. (Very minor)

**Pull request merged**